open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/entities/AbstractSubjectAreaEntity.java

```diff
@@ -16,6 +16,10 @@
 
 import java.util.List;
 
+/**
+ * @param <E> inherited from {@link Node} object type
+ * Abstract class for Subject Area client entity operations
```

Comment:
entity => node

Reply:
it is probably worth changing the folder from entities to nodes as well for consistency. You could do this in another issue if you like

Reply:
Okay.

Reply:
I think we should change the comment from entity to Node -even if we are changing the file names in another issue

Reply:
Changed